### PR TITLE
feat: add `font-numeric` to all numeric usages

### PR DESCRIPTION
**Pull request by evgenyboxer**
# Description

Pon requested that we use the tabular (monospaced) for all numbers. I have tried to cover it all, hopefully I haven't missed :)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Other

## How should this be tested?

Please provide instructions so we can test. Please also list any relevant details for your test configuration.

- [ ] Pretty much every feature that has numbers should have the new `font-numeric` class.

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code where relevant, particularly in hard-to-understand areas
- [x] My changes generate no new console warnings
- [x] The base of this PR is `master` if hotfix, `develop` if not

### CODE REVIEW

File: src/assets/css/global/typography.css [hunk 1]
```diff
@@ -1,38 +1,54 @@
...
+.font-extralight,
+.font-light,
+.font-normal,
+.font-medium,
+.font-semibold,
+.font-bold,
+.font-extrabold,
+.font-black {
   font-weight: unset;
 }
```
Comment (evgenyboxer) [inline src/assets/css/global/typography.css:53]: looks like we didn't run prettier here?

File: src/assets/css/global/typography.css [hunk 2]
```diff
@@ -66,4 +82,9 @@ th {
...
 .font-black {
   font-variation-settings: 'wght' 900;
-}
\ No newline at end of file
+}
+
+.font-numeric {
+  font-feature-settings: 'tnum';
+  letter-spacing: -0.5px;
```
Comment (evgenyboxer) [inline src/assets/css/global/typography.css:89]: These are the attributes Pon wanted to use for all numbers.

@garethfuller do you like the name `font-numeric`?

File: src/components/cards/TradeCard/GasReimbursement.vue [hunk 3]
```diff
@@ -7,9 +7,7 @@
     <div v-if="isActive()" class="message relative px-2 py-3">
       <div class="ml-12">
         <h6 v-text="$t('highGasFees')" class="relative text-sm" />
-        <div class="relative text-sm text-gray-500">
-          {{ text }}
-        </div>
+        <div class="relative text-sm text-gray-500" v-html="text" />
```
Comment (evgenyboxer) [inline src/components/cards/TradeCard/GasReimbursement.vue:10]: That is absolutely cool to have HTML working like that with the simple `v-html` tag!

File: src/assets/css/global/typography.css (hunk 2 above)
Comment (garethfuller) [reply src/assets/css/global/typography.css:89]: Yeah lgtm

File: src/components/cards/TradeCard/GasReimbursement.vue (hunk 3 above)
Comment (garethfuller) [reply src/components/cards/TradeCard/GasReimbursement.vue:10]: If it's just text and not html in the variable `text` you can just use `v-text` instead.

**Review by garethfuller — APPROVED** [review]
Looks good, thanks!

**Pull request merged**